README.md

```diff
@@ -109,7 +112,7 @@ Then, install Pyro using the `pytorch-1.0` branch.
 pip install git+https://github.com/uber/pyro.git@pytorch-1.0
 ```
 
-or, with the `extras` dependency to run examples/tutorials.
+or, with the `extras` dependency to run the probabilistic models included in the `examples`/`tutorials` directories:
```

Comment:
While this is unrelated to your PR, and I am only noticing this now, it will be better to put the instructions `Install using pip` and `Install from source` to the section `Installing Pyro dev branch` (removing the section `Finally install pyro`) because this is more generally applicable to the `dev` branch too. You could then refer to the same section here and mention that they would just need to append `@pytorch-1.0`, but the remaining instructions are the same. I'm happy to make a change myself to your PR, if you add me as a contributor.

Reply:
Sounds great!
I believe you could push to my branch directly since I have ticked "allow edits from maintainers".

Reply:
Great, I'll push a change!